openmdao/solvers/linear/scipy_iter_solver.py

```diff
@@ -141,8 +141,6 @@ def _mat_vec(self, in_vec):
         scope_out, scope_in = system._get_scope()
         system._apply_linear([vec_name], self._rel_systems, self._mode, scope_out, scope_in)
 
-        # print('in', in_vec)
```

Comment:
Please stop removing my debug statements here. I use them all the time, including just last week. @channel The debug prints in this story don't show near the granularity that these prints do.